torch_tvm/operators.cpp

```diff
@@ -269,6 +317,9 @@ RegisterTVMOperator reg({
        auto out =
            tvm::relay::CallNode::make(op, bn_inputs, tvm::Attrs(attrs), {});
        TORCH_INTERNAL_ASSERT(node->outputs().size() == 1);
+       // Is this right? Seems strange to use tupleget node if we are returning before that.
```

Comment:
I'll need to double check this -- can you file it as an issue?

Reply:
@bwasti, sure.